hw01/hw01.py

```diff
@@ -88,4 +94,15 @@ def hailstone(n):
     1
     """
     "*** YOUR CODE HERE ***"
-
+    arr=[]
+    while n>1:
+        arr.append(n)
+        if n%2==0:
+            n=n//2
+        else:
+            n=3*n+1
+    else:
+        arr.append(1)
+    return arr
+result=hailstone(8)
```

Comment:
Do not need to add function call. The assignment only requires you to implement the function.
```suggestion

```

Reply:
have deleted function call